app/services/katello/product_content_finder.rb

```diff
@@ -26,7 +26,8 @@ def product_content
       structured_apt_roots = roots.where(:content_id => nil)
       if structured_apt_roots.any?
         deb_repos_query = Katello::Repository.where(root: structured_apt_roots)
-        deb_repos = match_environment ? deb_repos_query.where(content_view_version: versions, environment: consumable.environment) : deb_repos_query.where(:library_instance_id => nil)
+        environment = consumable.respond_to?(:environment) ? consumable.environment : consumable.content_view_environments.select(:environment_id).map(&:environment_id)
```

Comment:
```suggestion
        environment = consumable.respond_to?(:environment) ? consumable.environment : consumable.content_view_environments.pluck(:environment_id)
```

Is there a reason not to use the `pluck` method here?

Reply:
No, thank you :sweat_smile:

Reply:
@jeremylenz line 18 was added in c33da7a.
Is there a reason this uses `select(:attribute).map(&:attribute)` rather then just `pluck(:attribute)`?

Is there a szenario, where `content_view_environments` contains objects that do not have the attribute?

Reply:
Then we should also leave it at this point I guess.